.circleci/config.yml

```diff
@@ -56,6 +56,7 @@ commands:
             md5sum lib/Pipfile >> ~/python_cache_key.md5
             md5sum lib/test-requirements.txt >> ~/python_cache_key.md5
             md5sum lib/test-requirements-with-tensorflow.txt >> ~/python_cache_key.md5
+            md5sum lib/setup.py >> ~/python_cache_key.md5
```

Comment:
If `setup.py` changes an entry point (as happens in this PR), the venv needs to be rebuilt